profiler/src/main.cpp

```diff
@@ -85,7 +86,7 @@ enum class ViewShutdown { False, True, Join };
 static tracy::unordered_flat_map<uint64_t, ClientData> clients;
 static std::unique_ptr<tracy::View> view;
 static tracy::BadVersionState badVer;
-static uint16_t port = 8086;
+static uint16_t port = tracy::DEFAULT_BROADCAST_UDP_PORT;
```

Comment:
What is the point of making this a value defined elsewhere? (Where? I now have to actively seek for it.)

Reply:
I was extracting it because we use it in multicapture as well (3 times IIRC). As this was currently in a .cpp file of `profiler` I can't reference it from elsewhere.

 I put it in `TracyProtocol.hpp`, that felt the best location to store protocol-related configuration.

[I can revert this extraction and use the value directly if you prefer - but I was trying to avoid having multiple `8096` lying around]